### DISCUSSION TITLE
My Experience with Trying to Sponsor UTM

### DISCUSSION THREAD

**Discussion by bit2pixel** [utmapp/UTM#5031] [General]
Hi friends, 

I wanted to share my experience with trying to sponsor UTM.

I had a rough time finding out how to sponsor the project as I couldn't find anything related in the web page. I was about to give up until I found the sponsor page on GitHub by pure chance.

I think it would be a wonderful improvement to make the process of supporting the project much clearer and easier to find on the website to prevent losing potential sponsors. Perhaps putting it front and center would be helpful, or at least a button at the top of the page.

Thanks for all the incredible and hard work! 💙

**Comment by KendrickLamar2003** [comment]
It already is on the top..?
![3E9EB278-74C5-44A5-BFE1-16C6850BFCF4](https://user-images.githubusercontent.com/87071118/219432559-2f8250e7-8014-431c-bf6f-f536bdcfae10.jpeg)

**Comment by bit2pixel** [comment]
@swaggio7 I'm pretty sure the average user's interaction flow doesn't start with GitHub but goes something like this:

<img width="700" alt="Screenshot 2023-02-16 at 10 18 11 PM" src="https://user-images.githubusercontent.com/127606/219565317-36d79606-0c82-4915-b32c-1dc2379fe262.png">

<img width="700" alt="Screenshot 2023-02-16 at 10 18 24 PM" src="https://user-images.githubusercontent.com/127606/219565330-7643d9fd-5bc0-43b6-96db-07354e71f3cc.png">

**Comment by KendrickLamar2003** [comment]
Oh, completely forgot about their website.
